disdat/dockerize.py

```diff
@@ -116,42 +154,24 @@ def dockerize(pipeline_root,
         docker_context,
     ]
 
-    retval = _do_subprocess(rsync_command, cli)
+    retval = disdat.common.do_subprocess(rsync_command, cli)
     if retval: return retval
 
     # PIP: Overwrite pip.conf in the context.template in your repo if they have set the option,
     # else just create empty file.
     # At this time, the Dockerfile always sets the PIP_CONFIG_FILE ENV var to this file.
-    pip_file_path = os.path.join(docker_context, "pip.conf")
-    if disdat_config.parser.has_option(_MODULE_NAME, 'dot_pip_file'):
-        dot_pip_file = os.path.expanduser(disdat_config.parser.get(_MODULE_NAME, 'dot_pip_file'))
-        shutil.copy(dot_pip_file, pip_file_path)
-        print("Copying dot pip file {} into {}".format(dot_pip_file, docker_context))
-    else:
-        touch_command = [
-            'touch',
-            pip_file_path
-        ]
-        retval = _do_subprocess(touch_command, cli)
-        if retval: return retval
+    retval = _copy_in_dot_file(disdat_config, docker_context, "pip.conf", "dot_pip_file", cli)
+    if retval: return retval
 
     # ODBC: Overwrite the .odbc.ini in the context.template in your repo if the user set the option,
     # else just create empty file.
     # At this time, the Dockerfile always sets the ODBCINI var to this file.
-    odbc_file_path = os.path.join(docker_context,"odbc.ini")
-    if disdat_config.parser.has_option(_MODULE_NAME, 'dot_odbc_ini_file'):
-        dot_odbc_ini_file = os.path.expanduser(disdat_config.parser.get(_MODULE_NAME, 'dot_odbc_ini_file'))
-        shutil.copy(dot_odbc_ini_file, odbc_file_path)
-        print("Copying dot odbc.ini file {} into {}".format(dot_odbc_ini_file, odbc_file_path))
-    else:
-        touch_command = [
-            'touch',
-            odbc_file_path
-        ]
-        retval = _do_subprocess(touch_command, cli)
-        if retval: return retval
+    retval = _copy_in_dot_file(disdat_config, docker_context, "odbc.ini", "dot_odbc_ini_file", cli)
+    if retval: return retval
+
+    setup_file = os.path.join(pipeline_root, 'setup.py')
```

Comment:
Probably should assert os.path.exists(setup_file)

Reply:
Added check